docs/peering.md

```diff
@@ -195,3 +201,125 @@ Example successful response:
   }
 }
 ```
+
+### Withdraw Peer Liquidity
+
+This is a two-phase transaction, so the withdrawal needs to be created first and then posted (i.e. the Account Servicing Entity commits to the withdrawal). If the withdrawal is faulty, it can be voided rather than posted.
```

Comment:
>If the withdrawal is faulty

The idea is that the Account Servicing Entity can safely perform withdraws in their internal system before posting in Rafiki (and can void in Rafiki if it failed internally).

Reply:
Let me add that and then you can approve again. That is very clarifying.